README.md

```diff
@@ -1,154 +1,139 @@
-# Nerves.Runtime
+# Nerves Runtime
 
 [![Build Status](https://travis-ci.org/nerves-project/nerves_runtime.svg)](https://travis-ci.org/nerves-project/nerves_runtime.svg)
 [![Hex version](https://img.shields.io/hexpm/v/nerves_runtime.svg "Hex version")](https://hex.pm/packages/nerves_runtime)
 
-Nerves.Runtime is a core component of Nerves. It contains applications and
-libraries that are expected to be useful on all Nerves devices.  Here are some
-of its features:
+Nerves Runtime is a core component of Nerves.
+It contains applications and libraries that are expected to be useful on all Nerves devices.
+Here are some of its features:
 
-* Generic system and filesystem initialization (suitable for use with
-  [bootloader](https://github.com/nerves-project/bootloader))
+* Generic system and filesystem initialization (suitable for use with [bootloader](https://github.com/nerves-project/bootloader))
 * Introspection of Nerves system firmware and deployment metadata
-* A custom shell for debugging and running commands in a `bash` shell like
-  environment
+* A custom shell for debugging and running commands in a `bash`-like environment
 * Device reboot and shutdown
-* A small Linux kernel `uevent` application for capturing hardware change events
-  and more
+* A small Linux kernel `uevent` application for capturing hardware change events and more
 * More to come...
 
-The following sections describe the features in more detail. For even more
-information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
+The following sections describe the features in more detail.
+For even more information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
 
-## System initialization
+## System Initialization
 
-Nerves.Runtime provides an OTP application that can initialize the system when
-it is started. For this to be useful, Nerves.Runtime must be started before
-other OTP applications especially since non-Nerves-aware applications almost
-certainly won't know that they need this to work. Specific initialization is
-detailed in other sections. To set Nerves.Runtime up with `bootloader`, you will
-need to do the following (luckily this should become easier as this new
-functionality propagates through templates and examples.)
+Nerves Runtime provides an OTP application (`nerves_runtime`) that can initialize the system when it is started.
+For this to be useful, Nerves Runtime must be started before other OTP applications, since most will assume that the system is already initialized before they start.
+To set up Nerves Runtime to work with Bootloader, you will need to do the following:
```

Comment:
Please wrap the text. I believe that 72 characters is the normal margin on other projects, but anything that gets the text to read easily in a terminal is fine with me (<80 characters is probably best).